crates/samples/direct2d/src/main.rs

```diff
@@ -451,7 +451,7 @@ fn create_device() -> Result<ID3D11Device> {
 
 fn create_render_target(factory: &ID2D1Factory1, device: &ID3D11Device) -> Result<ID2D1DeviceContext> {
     unsafe {
-        let d2device = factory.CreateDevice(device.cast::<IDXGIDevice>()?)?;
+        let d2device = factory.CreateDevice(&device.cast::<IDXGIDevice>()?)?;
```

Comment:
Can this now be inferred? `&device.cast()?`

Reply:
Unfortunately not...